package.json

```diff
@@ -10,11 +10,19 @@
     "url": "git@github.com:BBVAEngineering/ember-cli-htmlbars-minifier.git"
   },
   "scripts": {
-    "pretest": "npm run check && npm run lint",
-    "test": "npm run mocha",
-    "lint": "node_modules/.bin/eslint '**/*.js'",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint --max-warnings 0 ./*.js lib tests",
     "check": "node_modules/.bin/check-dependencies",
-    "mocha": "node_modules/.bin/mocha --recursive tests"
+    "pretest": "npm run check && npm run lint:js",
+    "test": "npm run clean-coverage && npm run test:node",
+    "test:node": "nyc --reporter=html --reporter=json --reporter=json-summary --reporter=lcov --report-dir coverage --exclude tests mocha tests",
```

Comment:
You missed `--recursive` option (dont know if needed).

Reply:
fixed calling to "test:node"